### ISSUE TITLE
Oculomotor synkinesis

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6984]
**Preferred term label:**
Oculomotor synkinesis

**Synonyms**

**Definition (free text, please give PubMed ID)**
oculomotor synkinesis consist of elevation of the upper eyelid on attempted downward gaze or adduction, adduction of the eye on attempted upward or downward gaze, and constriction of the pupil on attempted adduction.
PMID: 18004962

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:619215   3/5 PMID:31211835

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
